src/views/PeopleSearch/components/SearchFieldList/index.tsx

```diff
@@ -139,6 +150,13 @@ const SearchFieldList = ({ onSearch }: Props) => {
   const [departments, setDepartments] = useState<string[]>([]);
   const [buildings, setBuildings] = useState<string[]>([]);
   const [halls, setHalls] = useState<string[]>([]);
+  const [userProvidedYear, setUserProvidedYear] = useState(new Date().getFullYear());
+  const [graduationYearRange, setGraduationYearRange] = useState<number[]>([
+    1889,
+    userProvidedYear,
+  ]);
```

Comment:
I think that this doesn't need to be it's own state value, since we're already storing the initial year and final year in the search params. You can just provide defaults when the search param values are empty strings.

Reply:
I could, instead of unsing the set function to change graduationYearRange to just separate straight the newValue, but then it would be hard to implement the visualization of the hands of the slider changing.
![image](https://github.com/gordon-cs/gordon-360-ui/assets/115328139/944b05f0-80e1-4adf-a116-cda2eb8c2d65)
![image](https://github.com/gordon-cs/gordon-360-ui/assets/115328139/b6ad7ebc-bb90-4a8e-b2ca-3615429be6dc)

Reply:
@EjPlatzer are you satisfied with leaving this as it is?